### ISSUE TITLE
Screen_view is always automatically tracked

### ISSUE THREAD

**Issue by waghD** [angular/angularfire#2466]
I disabled automatic screen_view tracking because i didn't want route url parameters in my screen_names so i tracked them myself. But the automatic screen_view events are still being tracked even though i removed the ScreenTrackingService from my ngModule and set the send_page_view flag to false. 

I still see the debug logs from screen_view tracking in my browser console and see them in my firebase console as well. 

### Version info

**Angular:**
9.1.0
**Firebase:**
7.14.2
**AngularFire:**
6.0.0
**Other (e.g. Ionic/Cordova, Node, browser, operating system):**
@ionic/angular: 5.1.1

### How to reproduce these conditions

**AppModule**

```
@NgModule({
  declarations: [AppComponent],
  imports: [
    BrowserModule,
    IonicModule.forRoot(),
    AppRoutingModule,
    ComponentsModule,
    HttpClientModule,
    AngularFireModule.initializeApp(environment.firebaseConfig),
    AngularFireAuthModule,
    AngularFireStorageModule,
    AngularFireFunctionsModule,
    AngularFirestoreModule,
    AngularFireMessagingModule,
    AngularFireAnalyticsModule,
    AngularFirePerformanceModule,
    BrowserAnimationsModule,
    MatMomentDateModule,
    UserIdleModule.forRoot({idle: 600, timeout: 300, ping: 120}),
    ServiceWorkerModule.register('/combined-worker.js', {enabled: environment.production}),
  ],
  providers: [
    {provide: ErrorHandler, useClass: ErrorHandlerService},
    StatusBar,
    SplashScreen,
    File,
    CallNumber,
    FileTransfer,
    FileOpener,
    FingerprintAIO,
    AnalyticsService,
    PerformanceMonitoringService,
    UserTrackingService,
    {
      provide: CONFIG, useValue: {
        send_page_view: false,
        allow_ad_personalization_signals: false,
        anonymize_ip: true,
      }
    },
    {provide: DEBUG_MODE, useValue: environment.analyticsDebug},
    {provide: APP_NAME, useValue: environment.appName},
    {provide: APP_VERSION, useValue: environment.version},
    {provide: RouteReuseStrategy, useClass: IonicRouteStrategy},
    {provide: REGION, useValue: 'europe-west1'},
    {provide: LOCALE_ID, useValue: 'de-AT'},
    {provide: APP_INITIALIZER, useFactory: initializePwa, deps: [PwaService], multi: true},
    {provide: APP_INITIALIZER, useFactory: initializeTheme, deps: [ThemeService], multi: true},
  ],
  bootstrap: [AppComponent],
})
export class AppModule {
}
```

### Debug output
** Screenshots **
![image](https://user-images.githubusercontent.com/33834105/82438809-ecb24000-9a99-11ea-94cd-800f9224849e.png)

### Expected behavior

Angular Fire does not track and send screen_view events to firebase

### Actual behavior

AngularFire automatically tracks and sends screen_view events to firebase regardless of setup

**Comment by jamesgroat** [comment]
Workaround to disable this -- just provide a dummy class for the service:

`{      provide: ScreenTrackingService,      useValue: {},     },`

**Comment by jamesdaniels** [comment]
Addressed in `6.0.4`, sorry about that
